src/platform-implementation-js/dom-driver/gmail/views/gmail-nav-item-view.ts

```diff
@@ -66,6 +66,7 @@ export type NavItemDescriptor = {
     | DropdownButtonAccessoryDescriptor
     | null;
   type: keyof NavItemTypes;
+  tooltipAlignment: 'left' | 'top' | 'right' | 'bottom' | null;
```

Comment:
Using explicit `null` would require TS callers to update call sites.

```suggestion
  tooltipAlignment?: 'left' | 'top' | 'right' | 'bottom' | null;
```

Reply:
The property is within `Partial` type, so, it's already optional as all other properties (except the name), no changes will be required for existing users.
https://github.com/InboxSDK/InboxSDK/blob/02657eb133f1e7ed14612be63d3a2f16d5e03fa1/src/platform-implementation-js/dom-driver/gmail/views/gmail-nav-item-view.ts#L51-L70